src/rulesets/oas2/__tests__/valid-example.ts

```diff
@@ -154,8 +154,8 @@ describe('valid-example', () => {
     expect(results).toEqual([
       expect.objectContaining({
         code: 'valid-example',
-        message: '"halRoot" property type should be array',
-        path: ['definitions', 'halRoot'],
+        message: '"self" property type should be array',
+        path: ['definitions', 'halRoot', '_links', 'self'],
```

Comment:
Urgh, it was stupid of me. We did have a proper path here I erroneously changed in a different PR. It's now on par with [master](https://github.com/stoplightio/spectral/blob/master/src/rulesets/oas2/__tests__/valid-example.ts#L155L159) and should stay so.